server/controllers/inventory/inventory/core.js

```diff
@@ -78,14 +78,24 @@ function updateItemsMetadata(record, identifier) {
   if (record.group_uuid) {
     record.group_uuid = db.bid(record.group_uuid);
   }
+ 
 
   const sql = 'UPDATE inventory SET ? WHERE uuid = ?;';
+  // if there is no property to update this query won't work
+
+  if(_.isEmpty(record)) { //there is no change, but user has submitted
+    return getItemsMetadataById(identifier);
+  }else{
```

Comment:
This is much better.    ESLint should tell you that you do not need to `else` statement.  Please make sure you have ESLint installed in your editor.

```js
if (_.isEmpty(record)) {
  return getItemsMetadataById(identifier);
}

return db.exec(sql, [record, db.bid(identifier)])
  .then(() => getItemsMetadataById(identifier));
```